Sources/MapboxDirections/Intersection.swift

```diff
@@ -319,8 +323,9 @@ extension Intersection: Codable {
             approachLanes = lanes.map { $0.indications }
             usableApproachLanes = lanes.indices { $0.isValid }
             preferredApproachLanes = lanes.indices { ($0.isActive ?? false) }
-            let usableIndications = lanes.compactMap { $0.validIndication }
-            usableLaneIndication = usableIndications.reduce(LaneIndication(rawValue: 0)) { $0.union($1) }
+            usableLaneIndication = lanes.compactMap { $0.validIndication }[0]
+//            let usableIndications = lanes.compactMap { $0.validIndication }
+//            usableLaneIndication = usableIndications.reduce(ManeuverDirection(rawValue: "none")) { $0.union($1) }
```

Comment:
This introduces a hard assumption that the valid indications of each lane will always match. We’re OK with that assumption, but if the Directions API were to ever violate the assumption, perhaps we should throw `EncodingError.invalidValue(_:_:)` instead of silently swallowing the inconsistency. To check for an inconsistency:

```swift
Set(lanes.compactMap { $0.validIndication }).count > 1
```

Reply:
That's a good idea! I'll add this check